yt/data_objects/static_output.py

```diff
@@ -941,6 +941,8 @@ def _create_unit_registry(self):
         self.unit_registry.add("code_length", 1.0, dimensions.length)
         self.unit_registry.add("code_mass", 1.0, dimensions.mass)
         self.unit_registry.add("code_density", 1.0, dimensions.density)
+        # Note that energy is actually energy per unit mass
+        self.unit_registry.add("code_energy", 1.0, dimensions.energy / dimensions.mass)
```

Comment:
Is `code_energy` a good name? I'm worried that it's a little confusing since actually it's energy per unit mass. What do others think?

Reply:
We should call this `code_specific_energy`. More wordy, but more accurate.

Reply:
Agree!

Reply:
But when I search in the codebase, I see the name "energy_units" already used for what would be specific energy in enzo_p. Shall we keep consistent with the convention in enzo_p or use the more accurate name?

Reply:
Ping @ngoldbaum

Reply:
I agree with John, let’s use the more accurate name.